### ISSUE TITLE
fix Lighthouse CI errors

### ISSUE THREAD

**Issue by simpixelated** [simpixelated/simpixelated.com#15]
Fix errors:
- `non-composited-animations` [solution](https://developers.google.com/web/fundamentals/performance/rendering/stick-to-compositor-only-properties-and-manage-layer-count)
- `offscreen-images`
- `uses-rel-preconnect`

---

previous issue (now fixed)

enable disabled rules and fix errors

current rules disabled:
```js
        "color-contrast": "off",
        "errors-in-console": "off",
        "installable-manifest": "off",
        "maskable-icon": "off",
        "offline-start-url": "off",
        "service-worker": "off",
        "splash-screen": "off",
        "themed-omnibox": "off",
        "unused-javascript": "off",
        "uses-responsive-images": "off",
        "works-offline": "off",
        "tap-targets": "off",
```

**Comment by simpixelated** [comment]
Also consider switching to https://github.com/marketplace/actions/lighthouse-ci-action

**Comment by simpixelated** [comment]
looks to have been fixed by #34
